### DISCUSSION TITLE
Molecular ionisation

### DISCUSSION THREAD

**Discussion by keithbennett** [epochpic/epoch#122] [General]
In GitLab by Charles Varin on 07 Apr 2014, 16:07  (GitLab issue GL#223)

Hi,

right now, EPOCH implements ionisation with atomic models (Keldysh, ADK, ...). But the ionisation rates for atoms and molecules can differ considerably. Assuming that many laser-plasma experiments are done with molecular gases (unless a rare gas is used), modelling atomic rates isn't right and can give incorrect results. This is true for air, where atomic models predicts preferably ionisation of nitrogen, while molecular models and experiments show that oxygen is mainly ionised. Using molecular ionisation potentials is not enough.

There exists extensions to Keldysh and ADK models for the ionisation of diatomic molecules. For example, see section 2.3 in

Couairon, A., & Mysyrowicz, A. (2007). Femtosecond filamentation in transparent media. Physics Reports, 441(2-4), 47–189. doi:10.1016/j.physrep.2006.12.005

and

Mishima, K., Hayashi, M., Yi, J., Lin, S., Selzle, H., & Schlag, E. (2002). Generalization of Keldysh’s theory. Physical Review A, 66(3), 033401. doi:10.1103/PhysRevA.66.033401

Tong, X. M., Zhao, Z. X., & Lin, C. D. (2002). Theory of molecular tunneling ionization. Physical Review A, 66(3), 033402. doi:10.1103/PhysRevA.66.033402

Adding a "form = molecular/atomic" to the species block where "ionisation_energies" is called could allow switching back and forth between the molecular/atomic ionisation models.

Are there any plans to extend EPOCH capabilities in that area?

Regards,

Charles

**Comment by keithbennett** [comment]
In GitLab by Keith Bennett (@keithbennett) on 07 Apr 2014, 16:17 


Hi,

The PhD student who implemented the ionisation routines has finished and moved elsewhere. We have no plans to extend the existing routines ourselves but are happy to accept contributions.

Keith.

**Comment by keithbennett** [comment]
In GitLab by Charles Varin on 08 Apr 2014, 21:23 


Hi Keith,

thank you for your answer. We are planning a development phase for a new project and we will include molecular ionisation. If it works well, I contact you to know how to proceed for a potential merge (patch).

Regards,

 

Charles

**Comment by keithbennett** [comment]
In GitLab by Keith Bennett (@keithbennett) on 09 Apr 2014, 12:44 


Hi,

Okay, thanks. Any contributions are greatly appreciated.

Keith.

**Comment by keithbennett** [comment]
In GitLab by sassan on 25 Feb 2016, 12:00 UTC

there is an alternative way to simulate ionization of molecular O2 and N2. refer to PHYSICAL REVIEW A, 66, 033402 (2002). in this ref, you know how to deal with mplecular ionization of O2 and N2.

**Comment by keithbennett** [comment]
In GitLab by sassan on 25 Apr 2016, 11:04 

Hi Keith

i have a plan to add molecular ionization to EPOCH.

**Comment by keithbennett** [comment]
In GitLab by sassan on 25 Apr 2016, 11:04 

Hi Keith

i have a plan to add molecular ionization to EPOCH.

**Comment by keithbennett** [comment]
In GitLab by Keith Bennett (@keithbennett) on 25 Apr 2016, 11:37 

Reassigned to @sassan72

**Comment by keithbennett** [comment]
In GitLab by Keith Bennett (@keithbennett) on 25 Apr 2016, 13:23 

Okay, great. I've put it on the TODO milestone. This is where I put all tasks which we want to do but don't have any timescale for.

**Comment by keithbennett** [comment]
In GitLab by Heather Ratcliffe (@hratcliffe) on 18 Jun 2018, 13:03 

added ~1154 label

**Comment by keithbennett** [comment]
In GitLab by Joseph Smith on 07 Jul 2020, 18:05 

mentioned in issue GL#2216

**Comment by DanRight** [comment]
In what state collisional  ionization  model for molecules and atoms/ions is now?
